### PR TITLE
Mend CI with resource identifiers, increased timeout.

### PR DESCRIPTION
**Pull request by yashtewari**
The timeout bump is a workaround until we figure out why some findings are taking a long time to show up in the ES index after [this change](https://github.com/elastic/cloudbeat/commit/3c17c16c80c4ab6b78efa6bfaed8e9ec90fcecb2).

### CODE REVIEW

**Comment by oren-zohar** [comment]
@yashtewari I think we can backport this change

**Comment by oren-zohar** [comment]
@yashtewari let's backport it to 8.5.0

File: tests/product/tests/test_process_kubelet_rules.py [hunk 1]
```diff
@@ -46,11 +46,20 @@ def test_process_kubelet(elastic_client,
     # TODO: Implement a more optimal way of waiting
     time.sleep(60)
 
+    def identifier(eval_resource):
+        # Needs to be done because EKS findings are showing up in vanilla K8S tests,
+        # leading to unexpected findings structure.
```
Comment (oren-zohar) [inline tests/product/tests/test_process_kubelet_rules.py:51]: please open a ticket for it and link it

**Review by oren-zohar — CHANGES_REQUESTED** [review]

**Review by gurevichdmitry — APPROVED** [review]
@yashtewari great job, approved.
Please update refactor task to fix all functions doc strings.

**Review by oren-zohar — APPROVED** [review]
🥇

**Pull request merged**